code/game/gamemodes/blob/blobs/heatsink.dm

```diff
@@ -0,0 +1,24 @@
+/obj/effect/blob/heatsink
+	name = "heatsink blob"
+	icon_state = "blank_blob"
+	desc = "A fleshy mass capable of asborbing laser fire."
```

Comment:
How about "a brightly glowing red mass", to be consistent with the other blob pieces? If the sprite had long, fin-like tendrils (similar to a real heat sink), you could describe it that way as well.